Pmods/PmodCDC1/AD7156.c

```diff
@@ -174,11 +174,20 @@ void AD7156_BurstWriteReg(int reg, char size, int txData)
 {
 	unsigned char txBuffer[4] = {0x00, 0x00, 0x00, 0x00};
 
-	txBuffer[0] = (txData & (0xFF << 8 * (size - 1))) >> (8 * (size - 1));
-	txBuffer[1] = (txData & (0xFF << 8 * (size - 2))) >> (8 * (size - 2));
-	txBuffer[2] = (txData & (0xFF << 8 * (size - 3))) >> (8 * (size - 3));
-	txBuffer[3] =  txData & (0xFF << 8 * (size - 4));
-
+	switch(size)
+	{
+		case 4:
+			txBuffer[3] =  txData & (0xFF << 8 * (size - 4));
```

Comment:
i'd add a `/* FALLTHROUGH */` comment to each case ; 
since GCC 7, it can complain [give a warning or error] that you may have missed a `break` statement ; and they fixed it by requiring to add a `/* FALLTHROUGH */` comment;